### PR TITLE
Banana-666 - Change Captions

### PR DESCRIPTION
**Pull request by ECE372group203**
##SUMMARY
Add some stuff to learn how to use GIT

##REVIEWER
Don't need no review 'cuz it's perfect

### CODE REVIEW

**Review by CaffeineHelps — COMMENTED** [review]
With all due respect, this looks like complete shit.

**Comment by ECE372group203** [comment]
:koala:

File: _posts/0000-01-04-ECE372group203.md [hunk 1]
```diff
@@ -1,9 +1,10 @@
 ---
 layout: slide
-title: "Welcome to our slide deck!"
+title: "Ant's Slide deck, I suppose!"
 ---
 
-Caption here
+HelloWorld from Ant
+Don't speak to me like that ever again.
```
Comment (CaffeineHelps) [inline _posts/0000-01-04-ECE372group203.md:7]: Nobody makes me bleed my own blood

Comment (CaffeineHelps) [reply _posts/0000-01-04-ECE372group203.md:7]:
```suggestion
Always speak to me like that. 
```

**Review by CaffeineHelps — APPROVED** [review]
`THIS IS AMAZING SUCH WOW :+1: :cool:

File: _posts/0000-01-04-ECE372group203.md (hunk 1 above)
Comment (ECE372group203) [reply _posts/0000-01-04-ECE372group203.md:7]: Nope

**Pull request merged**